docs/ref/models/querysets.rst

```diff
@@ -8,12 +8,18 @@ Supported ``QuerySet`` methods
 All of Django's :doc:`QuerySet methods <django:ref/models/querysets>` are
 supported, except:
 
-  - :meth:`extra() <django.db.models.query.QuerySet.extra>`
-  - :meth:`prefetch_related() <django.db.models.query.QuerySet.prefetch_related>`
+- :meth:`~django.db.models.query.QuerySet.difference`
```

Comment:
MongoDB has `$setDifference` and `$setIntersection`. Not sure if they could be used to emulate. @WaVEV Did this cross your radar when implementing support for `union()`?

Reply:
I created INTPYTHON-823 Add support for QuerySet.intersection() and difference().